Content.Client/Administration/AdminNameOverlay.cs

```diff
@@ -1,23 +1,31 @@
+using System.Linq;
 using System.Numerics;
 using Content.Client.Administration.Systems;
+using Content.Shared.CCVar;
+using Content.Shared.Mind;
 using Robust.Client.Graphics;
 using Robust.Client.ResourceManagement;
 using Robust.Client.UserInterface;
-using Robust.Shared;
-using Robust.Shared.Enums;
 using Robust.Shared.Configuration;
+using Robust.Shared.Enums;
+using Robust.Shared.Prototypes;
 
 namespace Content.Client.Administration;
 
 internal sealed class AdminNameOverlay : Overlay
 {
     private readonly AdminSystem _system;
+    private readonly IConfigurationManager _config = IoCManager.Resolve<IConfigurationManager>(); // TODO is this even legal?
```

Comment:
Not sure why, but as a dependency the manager always comes up NULL so I can't read the cvar. Might have to look around because I vaguely recall that there was some other way to get cvar updates, hopefully it does not require EntitySystem stuff